config/pipeline.yaml

```diff
@@ -1318,33 +1318,38 @@ platforms:
     <<: *arm-device
     mach: broadcom
     dtb: dtbs/bcm2836-rpi-2-b.dtb
+    compatible: ['raspberrypi,2-model-b', 'brcm,bcm2836']
```

Comment:
Why do these values need to be hard-coded?

Reply:
They are extracted once from dtb when we add platform.
Otherwise we will need to fetch dts file (or extract from dtb), parse it in pipeline service for each device, and all that for same value every time.
In rare cases they might be updated, but also it is better if we are aware about it.

Reply:
Thanks for clarifying.